haskell/src/Monocle/Backend/Index.hs

```diff
@@ -222,6 +227,17 @@ instance ToJSON ChangesIndexMapping where
             ]
       ]
 
+createChangesIndex :: MServerName -> MIndexName -> IO (BH.BHEnv, BH.IndexName)
+createChangesIndex serverUrl indexName = do
+  let indexSettings = BH.IndexSettings (BH.ShardCount 1) (BH.ReplicaCount 0)
+  bhEnv <- Q.mkEnv serverUrl
+  let index = BH.IndexName indexName
```

Comment:
It might be safer to use the `BH.IndexName` newtype directly:
```suggestion
createChangesIndex :: MServerName -> BH.InderName -> IO (BH.BHEnv, BH.IndexName)
createChangesIndex serverUrl index = do
  let indexSettings = BH.IndexSettings (BH.ShardCount 1) (BH.ReplicaCount 0)
  bhEnv <- Q.mkEnv serverUrl
```